commit f6c101eac5c3428750638ce6bf3eda8522d75a8b (refs/changes/42/542/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-28T17:43:08-07:00 (3 years ago)
    
    IMGAPI-588 Update sdcnode version for OpenSSL 1.0.2j/1.0.1u vulnerability fixes
    Reviewed by: Robert Mustacchi <rm@joyent.com>

diff --git a/Makefile b/Makefile
index 1e5cce7..2c28c42 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ SMF_MANIFESTS = $(shell ls smf/manifests/*.xml)
 NODEUNIT	:= ./node_modules/.bin/nodeunit
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v0.12.15
+NODE_PREBUILT_VERSION=v0.12.16
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
